sdk/appconfiguration/azure-data-appconfiguration/src/samples/java/com/azure/data/appconfiguration/ConfigurationClientJavaDocCodeSnippets.java

```diff
@@ -274,21 +275,27 @@ public void deleteSetting() {
      */
     public void listSettings() {
         ConfigurationClient configurationClient = createSyncConfigurationClient();
-        // BEGIN: com.azure.data.applicationconfig.configurationclient.listSettings#SettingSelector
-        for (ConfigurationSetting setting : configurationClient.listSettings(new SettingSelector()
-            .keys("prodDBConnection"))) {
-            System.out.printf("Key: %s, Value: %s", setting.key(), setting.value());
-        }
-        // END: com.azure.data.applicationconfig.configurationclient.listSettings#SettingSelector
+        // BEGIN: com.azure.data.applicationconfig.configurationclient.listSettings#settingSelector
+        SettingSelector settingSelector = new SettingSelector().keys("prodDBConnection");
+        PagedIterable<ConfigurationSetting> csStream =  configurationClient.listSettings(settingSelector);
+        csStream.streamByPage().forEach(setting -> {
+            System.out.printf("Key: %s, Value: %s", setting.value().get(0).key(), setting.value().get(0).value());
+        });
+        // END: com.azure.data.applicationconfig.configurationclient.listSettings#settingSelector
+    }
 
-        /**
-         * Generates code sample for using {@link ConfigurationClient#listSettings(SettingSelector, Context)}
-         */
+    /**
+     * Generates code sample for using {@link ConfigurationClient#listSettings(SettingSelector, Context)}
+     */
+    public void listSettingsContext() {
+        ConfigurationClient configurationClient = createSyncConfigurationClient();
```

Comment:
From those design docs, this should be:
`com.azure.data.applicationconfig.configurationclient.listSettings#settingSelector-context` And probably there are a few below too.